datarobot_batch_scoring/main.py

```diff
@@ -252,12 +254,18 @@ def main(argv=sys.argv[1:]):
     skip_row_id = parsed_args['skip_row_id']
     output_delimiter = parsed_args.get('output_delimiter')
 
+    if pid and not lid:
```

Comment:
I think we should be able to do something like 
```batch_scoring --host=<host> --user=<user> {<project_id> <model_id> | --import_id=<ammaidentifier> } <dataset_filepath> --datarobot_key=<datarobot_key> {--password=<pwd> | --api_token=<api_token>}```
with either [subcommands](https://docs.python.org/2.7/library/argparse.html#sub-commands)  or [mutual-exclusion](https://docs.python.org/2.7/library/argparse.html#mutual-exclusion).

The best thing to do might be to make  `main,py`  much dumber. Have it only accept the arguments and then pass them into a function like `validate`. `validate` could do all of the checking that is now performed by `main.py`. The advantage of this is that it would make it much easier to use batch_scoring as a module.